modules/imgproc/test/test_approxpoly.cpp

```diff
@@ -377,4 +377,46 @@ TEST(Imgproc_ApproxPoly, bad_epsilon)
     ASSERT_ANY_THROW(approxPolyDP(inputPoints, outputPoints, eps, false));
 }
 
+TEST(Imgproc_ApproxPoly, boundingPolygon)
+{
+    string imgPath = cvtest::findDataFile("imgproc/approxBoundingPoly.png");
```

Comment:
I propose to replace the test image with something synthesized in code with drawing functions.

Reply:
I might suggest replacing the test image with a 32 point image using **cv::polylines**. 
Should I reduce/increase the number of points or do something else?
![image](https://github.com/opencv/opencv/assets/98156294/5327c85a-4d5b-4cbc-9e81-afc59e234439)
Should I close the PR in opencv_extra in this case?

Reply:
Looks good! Thanks!